src/vector_of_array.jl

```diff
@@ -135,6 +135,12 @@ function Base.Array{U}(VA::AbstractVectorOfArray) where {U}
     vecs = vec.(VA.u)
     Array(reshape(reduce(hcat, vecs), size(VA.u[1])..., length(VA.u)))
 end
+
+function Base.convert(::Type{AbstractArray}, VA::AbstractVectorOfArray)
+    vecs = vec.(VA.u)
+    return reshape(stack(vecs), size(VA.u[1])..., length(VA.u))
```

Comment:
why is the reshape necessary after a stack? I thought that function was supposed to take care of that part?

Reply:
my bad. fixed